src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ImplementSerializationConstructorsTests.cs

```diff
@@ -171,65 +170,7 @@ End Sub
                         throw new NotImplementedException()
                     End Sub
                 End Class",
-                GetCA2229BasicResultAt(8, 32, "CA2229HasConstructorWrongAccessibility", CA2229MessageUnsealed));
-        }
-
-        [Fact]
-        public async Task CA2229HasConstructorWrongAccessibilityWithScope()
```

Comment:
😕 Where did this test go?

Reply:
I forgot to mention that I have removed this test that was duplicating the previous one to use scope but no longer works as there are now multiple rules with the same ID. Is it ok to do so?

Reply:
I think it is fine to remove duplicate tests with "WithScope" suffix. Long back we had such duplicate tests for scope validation, but they were mostly removed, guess some were left behind.